workflow/rocoto/gfs_tasks.py

```diff
@@ -950,10 +950,6 @@ def _fcst_cycled(self):
             dependencies.append(rocoto.add_dependency(dep_dict))
             dependencies = rocoto.create_dependency(dep_condition='or', dep=dependencies)
 
```

Comment:
There are 2 blank lines here.  This is likely causing the `pynorms` test to fail